py-scripts/real_application_tests/teams_automation/lf_interop_teams.py

```diff
@@ -0,0 +1,990 @@
+                            video.get("vi_sent_bitrate", 0),
+                            video.get("vi_recv_bitrate", 0),
+                            video.get("vi_sent_frame_rate", 0),
+                            video.get("vi_sent_res", "NA"),
+                            video.get("vi_rtt", 0),
+                            video.get("vi_sent_pkts", 0),
+                            video.get("vi_sent_codec", "NA"),
+                            video.get("vi_processing", "NA"),
+                        ]
+                    writer.writerow(row)
+
+            return jsonify({"status": "success"}), 200
+
+        try:
+            self.app.run(host='0.0.0.0', port=5005, debug=True, threaded=True, use_reloader=False)
+        except Exception as e:
+            logging.info(f"Error starting Flask server: {e}")
+            sys.exit(0)
+
+    def create_avg_data(self):
+        output_file = os.path.join(self.path, "teams_call_avg_data.csv")
+        summary_rows = []
+
+        for csv_path in glob.glob(os.path.join(self.path, "*.csv")):
+            if csv_path.endswith("teams_cred.csv"):
+                continue
+            df = pd.read_csv(csv_path)
+
+            device_name = os.path.splitext(os.path.basename(csv_path))[0]
+            df = df.drop(columns=["timestamp"], errors="ignore")
+
+            numeric_cols = df.select_dtypes(include="number").columns
+            averages = df[numeric_cols].mean().round(2)
+
+            row = averages.to_dict()
+            row["Device Name"] = device_name
+            summary_rows.append(row)
+
+        summary_df = pd.DataFrame(summary_rows)
+
+        cols = ["Device Name"] + [col for col in summary_df.columns if col != "Device Name"]
+        summary_df = summary_df[cols]
+
+        summary_df.to_csv(output_file, index=False)
+        logger.info(f"Avg data saved to {output_file}")
+
+    def stop_test_in_webui(self):
+        try:
+            url = f"http://{self.lanforge_ip}:5454/update_status_yt"
+            headers = {
+                'Content-Type': 'application/json',
+            }
+
+            data = {
+                'status': 'Completed',
+                'name': self.test_name
+            }
+
+            response = requests.post(url, json=data, headers=headers)
+
+            if response.status_code == 200:
+                logging.info("Successfully updated STOP status to 'Completed'")
+                pass
+            else:
+                logging.error(f"Failed to update STOP status: {response.status_code} - {response.text}")
+
+        except Exception as e:
+            logging.error(f"An error occurred while updating status: {e}")
+
+
+def main():
+    try:
+
+        parser = argparse.ArgumentParser(
+            prog='lf_interop_teams.py',
+            formatter_class=argparse.RawTextHelpFormatter,
+            epilog=''' Allows user to run the Microsoft Teams Call test on a target resource for the given duration. ''',
+            description="""
```

Comment:
Will this printout tabbed in? I see most other scripts have the description left-justified w/ no whitespace between text and left side of editor.